bamboo/set-bamboo-env-variables.sh

```diff
@@ -140,7 +146,7 @@ fi
 # Target master by default.
 # Update with appropriate conditional
 # when creating a feature branch.
-export PR_BRANCH=master
+export PR_BRANCH="jk/CUMULUS-1-fix-bamboo-script"
```

Comment:
I assume this is not intentional?

Reply:
Correct, this was set to demonstrate the PR works.  Will revert.